src/api/authApi.ts

```diff
@@ -25,11 +26,15 @@ export const getAccessToken = async () => {
   }
   try {
     const currentSession = await fetchAuthSession();
-    return currentSession.tokens?.accessToken.toString() ?? null;
-  } catch (error) {
-    if (error === 'The user is not authenticated') {
-      window.location.href = UrlPathTemplate.Home;
+    if (currentSession.tokens) {
+      return currentSession.tokens.accessToken.toString();
+    } else {
+      const searchParams = new URLSearchParams();
+      searchParams.set(LocalStorageKey.RedirectPath, getCurrentPath());
```

Comment:
Kanskje litt rart, men syns det ble bedre å gjenbruke `LocalStorageKey.RedirectPath` også for URL param her, enn å lage en ny konstant som bare har denne key-en. Det som ligger her skal potensielt bare settes i localstorage i neste steg likevel, så gir mening at de har likt navn.